packages/@react-spectrum/list/src/RootDropIndicator.tsx

```diff
@@ -4,22 +4,25 @@ import {useVisuallyHidden} from '@react-aria/visually-hidden';
 
 export default function RootDropIndicator() {
   let {dropState, dropHooks} = useContext(ListViewContext);
-  let dropRef = useRef();
+  let ref = useRef();
   let {dropIndicatorProps} = dropHooks.useDropIndicator({
     target: {type: 'root'}
-  }, dropState, dropRef);
-
+  }, dropState, ref);
+  let isDropTarget = dropState.isDropTarget({type: 'root'});
   let {visuallyHiddenProps} = useVisuallyHidden();
-  if (dropIndicatorProps['aria-hidden']) {
+  
+  if (!isDropTarget && dropIndicatorProps['aria-hidden']) {
     return null;
   }
 
   return (
-    <div
-      role="option"
-      aria-selected="false"
-      {...visuallyHiddenProps}
-      {...dropIndicatorProps}
-      ref={dropRef} />
+    <div role="row" aria-hidden={dropIndicatorProps['aria-hidden']}>
+      <div
+        role="gridcell"
+        aria-selected="false"
+        {...visuallyHiddenProps}>
+        <div role="button" {...dropIndicatorProps} ref={ref} />
```

Comment:
Similar question here, are the `dropIndicatorProps` invalid on the row and thus need to be added to a button?